tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -47,8 +47,10 @@ def create_parser(product_choices=None):
 
 TEST is either the full path to a test file to run, or the URL of a test excluding
 scheme host and port.""")
-    parser.add_argument("--manifest-update", action="store_true", default=False,
+    parser.add_argument("--manifest-update", action="store_true",
```

Comment:
Your PR comment mentions setting the `default` to `None`. I assume this is happening implicitly, but elsewhere below we explicitly set `None` defaults. Do we have a stylistic preference?.